smoke-test.sh

```diff
@@ -0,0 +1,34 @@
+# grab go-fdo-client at revision (TODO(runcom): stick to a revision)
+git clone https://github.com/fido-device-onboard/go-fdo-client
+cd go-fdo-client
+go build -o fdo_client ./cmd/fdo_client
+cd ../
+rm -rf mfg.db rv.db own.db
+# start manufacturing server
+go-fdo-server serve 127.0.0.1:8038 --db ./mfg.db --db-pass 1234Cia0@! --debug &
+timeout 300 bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' 127.0.0.1:8038/health)" != "200" ]]; do sleep 5; done' || false
+# start owner server
+go-fdo-server serve 127.0.0.1:8041 --db ./rv.db --db-pass 1234Cia0@! --debug &
+timeout 300 bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' 127.0.0.1:8041/health)" != "200" ]]; do sleep 5; done' || false
+# start rendezvous server
+go-fdo-server serve 127.0.0.1:8043 --db ./own.db --db-pass 1234Cia0@! --debug &
+timeout 300 bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' 127.0.0.1:8043/health)" != "200" ]]; do sleep 5; done' || false
+# creating new rv info data works
+# but when onboarding, the rv throws a not found... TODO(runcom): investigate
```

Comment:
I think you need to run TO0 after uploading the ownership voucher to the owner server:
```
curl "http://localhost:8043/api/v1/to0/${GUID}"
```